rasa_core/policies/keras_policy.py

```diff
@@ -122,6 +202,10 @@ def _persist_configuration(self, config_file):
         utils.dump_obj_as_json_to_file(config_file, model_config)
 
     def persist(self, path):
+        # type: (Text) -> None
+
+        super(KerasPolicy, self).persist(path)
```

Comment:
i am not so sure about this style wise: in persist we use super implementation (but in load we do not).

rather put the `self.featurizer.persist(path)` into `if self.model` branch, as there must be a featurizer present right?